lib/config/presets/internal/monorepo.ts

```diff
@@ -275,6 +275,10 @@ const patternGroups = {
   babel6: '^babel6$',
   clarity: ['^@cds/', '^@clr/'],
   embroider: '^@embroider/',
+  'fabric8-kubernetes-client': [
+    'io.fabric8:kubernetes-',
+    'io.fabric8:openshift-',
+  ],
```

Comment:
i would prefer to group by source url if possible

Reply:
I created a simple repository to reproduce - https://github.com/rombert/renovate-kubernetes-test . The logs indicate that all the dependencies are up-to-date, but I know for the Kubernetes Java client there should be a version bump from 6.7.1 to 6.7.2 .

In the logs ( https://developer.mend.io/github/rombert/renovate-kubernetes-test/-/job/8d4bbff3-1b44-4d11-b4ae-661b10b71a7d )  I see something like

```
          {
            "datasource": "maven",
            "depName": "io.fabric8:kubernetes-model-core",
            "currentValue": "6.7.1",
            "updates": [],
            "packageName": "io.fabric8:kubernetes-model-core",
            "skipReason": "disabled"
          }
```

These dependencies come from the osgi dependency manager that I contributed ( https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/osgi/index.ts ) . Is this something that is missing from there?

Reply:
it's because you disabled all osgi packages.

you can't use sourceUrl matching to enable packages. it's to late. but you can and should use sourceUrl matching for grouping.